reference-implementation/run-web-platform-tests.js

```diff
@@ -33,9 +33,15 @@ async function main() {
 
   const filterGlobs = process.argv.length >= 3 ? process.argv.slice(2) : ['**/*.html'];
   const excludeGlobs = [
+    'piping/general-addition.any.html', // FIXME: reenable this test as part of https://github.com/whatwg/streams/issues/1243.
     // These tests use ArrayBuffers backed by WebAssembly.Memory objects, which *should* be non-transferable.
     // However, our TransferArrayBuffer implementation cannot detect these, and will incorrectly "transfer" them anyway.
-    'readable-byte-streams/non-transferable-buffers.any.html'
+    'readable-byte-streams/non-transferable-buffers.any.html',
+    'readable-streams/garbage-collection.any.html', // FIXME: reenable this test once server provides common/gc.js.
```

Comment:
This should be fixed in [wpt-runner 5.0.0](https://github.com/domenic/wpt-runner/releases/tag/v5.0.0), let me check.